FHIR-SW-Development-Guide.ipynb

```diff
@@ -0,0 +1,1560 @@
+      "     [xslt] Loading stylesheet /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/template/scripts/onGenerate.final.xslt\n",
+      "\n",
+      "onGenerate.igQa:\n",
+      "     [xslt] Processing /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/template/onGenerate-ig-updated.xml to /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/template/ongenerate-validation-igqa.json\n",
+      "     [xslt] Loading stylesheet /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/template/scripts/onGenerate.qa.xslt\n",
+      "\n",
+      "onGenerate.artifactList:\n",
+      "     [xslt] Processing /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/template/onGenerate-ig-updated.xml to /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/temp/pages/_includes/artifacts.xml\n",
+      "     [xslt] Loading stylesheet /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/template/scripts/createArtifactSummary.xslt\n",
+      "\n",
+      "onGenerate.extend:\n",
+      "\n",
+      "onGenerate:\n",
+      "Generating Summary Outputs                                                       (00:36.0511)\n",
+      "Sending Usage Stats to Server                                                    (00:39.0651)\n",
+      "Jekyll: Source: /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/temp/pages (00:40.0825)\n",
+      "Jekyll: Generating...                                                            (00:40.0825)\n",
+      "Jekyll: done in 0.76 seconds.                                                    (00:41.0585)\n",
+      "Installing ncpi#dev to the package cache\n",
+      "  Fetching:\n",
+      "  Installing:  done.\n",
+      "Checking Output HTML                                                             (00:42.0547)\n",
+      "found 569 files                                                                  (00:42.0553)\n",
+      "  ... 569 html files, 0 pages invalid xhtml (0%)                                 (00:48.0643)\n",
+      "  ... 57674 links, 0 broken links (0%)                                           (00:48.0643)\n",
+      "Build final .zip                                                                 (00:48.0643)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Final .zip built                                                                 (00:49.0788)\n",
+      "Finished. 00:28.0239. Validation output in /Users/singhn4/Projects/fhir-sci/ncpi-model-forge/site_root/output/qa.html\n",
+      "Errors: 0  Warnings: 6  Info: 9 (00:50.0183)\n",
+      "Done                                                                             (00:50.0184)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run the HL7 validation tool\n",
+    "!java -jar org.hl7.fhir.publisher.jar -ig site_root/ig.ini -tx n/a"
```

Comment:
Other developers might've not downloaded the publisher JAR, so why don't we add `wget https://storage.googleapis.com/ig-build/org.hl7.fhir.publisher.jar`?

Reply:
The notebook points to the README where it explains what Method 2 is and how to install publisher and get the right version.